CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.8.0] 2024-10-21
+
+### Multiplayer Use Cases
+
+#### Added
+- The last page of the "Data/Event Synchronization" tutorial redirects to the "Proximity Checks" tutorial
+- Imported TextMeshPro essentials so you don't have to do it
+
+#### Changed
+- Updated project to Unity 6 (and packages to the latest stable versions)
```

Comment:
It'd be helpful to highlight the relevant project dependencies that were updated with from & to version.

Reply:
Do we still need that? (I added a comment about this in the 1.8.0 release doc, we can have the conversation there).
In the meantime, to unblock the release, the full list here